web/src/services/storageService.ts

```diff
@@ -20,4 +20,8 @@ export class StorageService {
   public setUserGroup(group: Group): void {
     localStorage.setItem(this.groupKey, JSON.stringify(group));
   }
+
+  public tokenAvailability(): boolean {
```

Comment:
I prefer to prefix methods which return booleans with 'has-' 'is-'. This makes easier to determine what exactly code is checking. Probably `isTokenInStorage` or `isTokenExists` or `hasToken` would be more concise and not less descriptive.